pyQuARC/main.py

```diff
@@ -6,19 +6,20 @@
 
 from tqdm import tqdm
 
-from pyQuARC.code.utils import get_headers
-
 if __name__ == '__main__':
+    from code.utils import get_headers
```

Comment:
why twice?

Reply:
This appears to be a mistake. Lines 10 and 17 should be removed.